src/infrastructure/jira/jira-design-issue-property-service.ts

```diff
@@ -46,55 +46,71 @@ const ATTACHED_DESIGN_URL_V2_VALUE_SCHEMA: JSONSchemaTypeWithId<AttachedDesignUr
 	};
 
 export class JiraDesignIssuePropertyService {
-	/**
-	 * @throws {ForbiddenByJiraServiceError} The app does not have permission to edit the Issue.
-	 */
-	saveDesignUrlInIssueProperties = async (
+	trySaveDesignUrlInIssueProperties = async (
```

Comment:
**note:** Converted `trySaveDesignUrlInIssueProperties` to `saveDesignUrlInIssueProperties` by analogue with `figmaService.tryCreateDevResourceForJiraIssue` to highlight that the method makes the best effort to perform the task.